ccloud/beginner-cloud/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '2'
 services:
   connect-cloud:
-    image: cnfldemos/cp-server-connect-datagen:0.1.7-5.4.0-SNAPSHOT
+    image: cnfldemos/cp-server-connect-datagen:0.1.7-5.4.x-latest
```

Comment:
@ybyzek  @rspurgeon Is this right?

Reply:
👍  that's my thinking as well.  @rspurgeon do you agree?

Reply:
Yes, that works